.gitignore

```diff
@@ -15,19 +15,7 @@ script.js
 
 /versioninfo.json
 
-/vendor/**/.editorconfig
-/vendor/**/*.jpg
-/vendor/**/*.json
-/vendor/**/.*.json
-/vendor/**/Makefile
-/vendor/**/*.png
```

Comment:
I guess this is the only line that actually was problematic?

Reply:
Actually the first one (*.json)

Reply:
Given that we did not have (any known to me) problems for 3 years, I would kind of prefer to just fix the case that this doesn't work for you. 

There is not a single `.json` file here so I think just dropping that is way easier?

The alternative will be to actually readd all the known CI ones, MakeFile and the currently 2 big testdata files.

I am also okay with adding the line you did alongside these.

Reply:
> There is not a single .json file here so I think just dropping that is way easier?

I did this on the dashboard integration branch.
This is fine for me, and in this case let's reject this PR. I just thought potential future problems could be avoided.

Reply:
I also agree that if this has been like this for a long time, and now we only have a very specific need, then I'd just add that as an exception to the rule, instead of reverting wider condition 👍🏻

Reply:
OK, I close it.